plugins/PostProcessingPlugin/scripts/FilamentChange.py

```diff
@@ -19,6 +22,14 @@ def getSettingDataString(self):
             "version": 2,
             "settings":
             {
+                "marlin": 
+                {
+                    "label": "Use Marlin Config?",
+                    "description": "Use configuration from Marlin Advanced Pause Feature!",
```

Comment:
We should use a more... neutral tone.
```suggestion
                    "description": "Use configuration from Marlin's Advanced Pause Feature.",
```